setup/openchange_provision

```diff
@@ -97,5 +98,7 @@ if not opts.openchangedb:
     else:
         parser.print_help()
         sys.exit(1)
+elif opts.check:
```

Comment:
It does the check operation..
I have fixed this and other similar instances in #199